modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/test/js/CheckboxMultiple/CheckboxMultiple.es.js

```diff
@@ -18,6 +18,37 @@ const CheckboxMultipleWithProvider = (props) => (
 	</PageProvider>
 );
 
+describe('Smoke test', () => {
+	test('field Checkbox Multiple renders the expected structure with default props', () => {
+		const {container} = render(
+			<CheckboxMultipleWithProvider
+				accessibleProps={{'aria-required': false}}
+				disabled={false}
+				inline={false}
+				isSwitcher={false}
+				localizedValueEdited={false}
+				name="namePropertyValue"
+				options={[
+					{
+						label: 'Option1',
+						reference: 'Option1Reference',
+						value: 'Option1Value',
+					},
+					{
+						label: 'Option2',
+						reference: 'Option2Reference',
+						value: 'Option2Value',
+					},
+				]}
+				predefinedValue={[]}
+				value={[]}
```

Comment:
Why not add values to `value` and `predefinedValue`? Recently an engineer on the team had trouble with the following code snippet:

```
const [value, setValue] = useState(initialValue ?? predefinedValue);

useEffect(() => {
  if (initialValue?.length > 0) {
	  setValue(initialValue);
  }
}, [initialValue]);
```

I realized that the `'has a value'`  test does not cover this case, but this could also be covered there.

I would just like to ensure that when we have `value` and `predefinedValue` the value will be the value rendered and used in the hidden input.

What do you think?